packages/config/src/options.ts

```diff
@@ -181,260 +178,245 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
-    canUpdateDuringTestTime: false,
   }, {
     name: 'env',
     defaultValue: {},
     validation: validate.isPlainObject,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'execTimeout',
     defaultValue: 60000,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'experimentalFetchPolyfill',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'experimentalInteractiveRunEvents',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'experimentalSessionAndOrigin',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'experimentalModifyObstructiveThirdPartyCode',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'experimentalSourceRewriting',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'fileServerFolder',
     defaultValue: '',
     validation: validate.isString,
     isFolder: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'fixturesFolder',
     defaultValue: 'cypress/fixtures',
     validation: validate.isStringOrFalse,
     isFolder: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'excludeSpecPattern',
     defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? ['**/__snapshots__/*', '**/__image_snapshots__/*'] : '*.hot-update.js',
     validation: validate.isStringOrArrayOfStrings,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'includeShadowDom',
     defaultValue: false,
     validation: validate.isBoolean,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'keystrokeDelay',
     defaultValue: 0,
     validation: validate.isNumberOrFalse,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'modifyObstructiveCode',
     defaultValue: true,
     validation: validate.isBoolean,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'nodeVersion',
     validation: validate.isOneOf('bundled', 'system'),
-    canUpdateDuringTestTime: false,
   }, {
     name: 'numTestsKeptInMemory',
     defaultValue: 50,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
```

Comment:
Same with this opt...this shouldn't be able to update. 😕

Reply:
Maybe there's not a great reason to change this during a test, but it's still technically possible, right? I think that's the main consideration, whether you can instead of whether you should.